spec/AutomatedLinkAcceptance+services.yaml

```diff
@@ -93,170 +105,191 @@ clients:
 
   - http-client:
       application-name: RegistryOffice
-      release-number: 1.0.0
-      uuid: xx-1-0-0-http-c-bm-ro-1-0-0-000
+      release-number: 2.0.0
+      uuid: ALA-1-0-0-http-c-ro-2-0-0-000
     tcp-client:
-      ip-address: 10.118.125.157
-      tcp-port: 1000
-      uuid: xx-1-0-0-tcp-c-bm-ro-1-0-0-000
+      remote-protocol: http
+      ip-address: 1.1.3.8
+      tcp-port: 3008
+      uuid: ALA-1-0-0-tcp-c-ro-2-0-0-000
     operation-clients:
       own-oam:
         basic:
           - service-name: /v1/register-application
-            uuid: xx-1-0-0-op-c-bm-ro-1-0-0-000
+            uuid: ALA-1-0-0-op-c-bm-ro-2-0-0-000
           - service-name: /v1/relay-server-replacement
-            uuid: xx-1-0-0-op-c-bm-ro-1-0-0-001
+            uuid: ALA-1-0-0-op-c-bm-ro-2-0-0-001
           - service-name: /v1/deregister-application
-            uuid: xx-1-0-0-op-c-bm-ro-1-0-0-002
+            uuid: ALA-1-0-0-op-c-bm-ro-2-0-0-002
           - service-name: /v1/relay-operation-update
-            uuid: xx-1-0-0-op-c-bm-ro-1-0-0-003
+            uuid: ALA-1-0-0-op-c-bm-ro-2-0-0-003
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
+
```

Comment:
empty lines can be removed (also for the subsequent http clients)

Reply:
All unnecessary empty lines were removed